datasets/simulated.py

```diff
@@ -41,79 +41,119 @@ def __init__(
         img_size=32,
         n_classes=2,
         train_frac=0.8,
-        framework='pytorch',
+        val_frac=0.8,
```

Comment:
I would rename `train_frac`, as `train_val_frac` and `val_frac` to `train_frac`, in order to have a clearer picture